rest-api/src/main/resources/definitions/event_stream_adherence_report.yml

```diff
@@ -25,6 +25,10 @@ properties:
         $ref: ./day_range.yml
         readOnly: true
         x-nullable: false
+    progression:
+        $ref: ./enums/participant_study_progress.yml
```

Comment:
Should add "no_schedule" to the enum list in participant_study_progress.yml also.